WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/download/progress/BackupDownloadProgressViewModel.kt

```diff
@@ -99,7 +104,21 @@ class BackupDownloadProgressViewModel @Inject constructor(
                 _snackbarEvents.postValue(Event(GenericFailureMsg))
             }
             is Progress -> {
-                _snackbarEvents.value = Event(SnackbarMessageHolder((UiStringText("Progress = ${state.progress}"))))
+                (_uiState.value as? Content)?.let { content ->
+                    val updatedList = content.items.map { contentState ->
+                        if (contentState.type == BACKUP_PROGRESS) {
+                            contentState as ProgressState
+                            contentState.copy(progress = state.progress ?: 0,
+                                    label = UiStringResWithParams(
+                                            string.backup_download_progress_label,
+                                            listOf(UiStringText(state.progress?.toString() ?: "0")))
+                            )
```

Comment:
Nitpick(🔍): Might want to check method call formatting (line break after the opening parenthesis):
`contentState.copy(progress = state.progress ?: 0,`

Reply:
> Hope detekt gets integrated soon

Oh my gosh, so do I!. I run ktlint and code analyze, but detekt would be welcome. 👍

Reply:
in 35dcba4